app/uk/gov/hmrc/exports/controllers/request/ExportsDeclarationRequest.scala

```diff
@@ -34,7 +34,7 @@ case class ExportsDeclarationRequest(
   consignmentReferences: Option[ConsignmentReferences] = None,
   departureTransport: Option[DepartureTransport] = None,
   borderTransport: Option[BorderTransport] = None,
-  containerData: Option[TransportInformationContainers] = None,
+  transportData: Option[TransportData] = None,
```

Comment:
can we not suffix it with `data`

Reply:
Are you saying you'd prefer not to have the suffix?